### DISCUSSION TITLE
[M-Z] (12 Febbraio 2019) Esercizio 8

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#490] [Esami & Esoneri M-Z]
### Traccia

<img width="958" alt="image" src="https://github.com/user-attachments/assets/8bfdd7e9-338f-40fe-a3cc-1126a9ea4574">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190212.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190212.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/a0421a56-3a02-435b-8b4c-7c614d2a1026)
